app/lib/widgets/app_bar/app_bar.dart

```diff
@@ -61,26 +36,21 @@ class FlexAppBar extends StatelessWidget implements PreferredSizeWidget {
 
   @override
   Widget build(BuildContext context) {
-    final styles = _getThemeStyles(context);
-
     return AppBar(
-      automaticallyImplyLeading: false,
+      automaticallyImplyLeading: true,
```

Comment:
whats this? 🤔

Reply:
So by default, `AppBar` inserts a leading widget (back button) if the corresponding page is used with routing. This was initially disabled in `FlexAppBar`, which is an extension of `AppBar`. 

Enabling it now allows for the leading back button to appear

Reply:
When we route to a page with the AppBar or FlexAppBar, this will automatically add a back arrow or "X" depending on the route itself (and how we got there).

There's more complex functionality we can add to the FlexAppBar to handle routing, but this should work as a simple use case. If we notice issues during testing, let's flip it back and handle "back" functionality manually via the leadingIcon defaults.